homework/java/shelest_vladimir/login_server/src/main/java/by/jrr/service/IUser.java

```diff
@@ -0,0 +1,5 @@
+package by.jrr.service;
+
+public interface IUser {
+    boolean LogIn(String enteredLogin, String enteredPassword);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.